### PR TITLE
Autospader

### PR DESCRIPTION
**Pull request by gausie**
- **Allow multiple clients at the same time**
- **Move to got for cookie management baked into fetching**
- **Formatting**
- **Add autospading first attempt (only includes items for now)**

This is mostly to merge so that other things can happen on oaf without conflicts. There are some changes to kol.js but otherwise spading remains unchanged in its implementation (i.e. autospade is not used yet)

### CODE REVIEW

File: packages/oaf/src/commands/kol/spade.ts [hunk 1]
```diff
@@ -83,6 +87,7 @@ const ITEM_SPADING_CALLS = [
       [
         "inv_spleen.php",
         {
+          pwd: true,
```
Comment (horrible-little-slime) [inline packages/oaf/src/commands/kol/spade.ts:90]: wait, are these still plugged in? is this still how `/spade items` works?

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]
I think the password thing should legit be changed. 

On a related note, I think there's a chance that `output.txt` is intended to be in gitignore? Just by vibes alone. Ignore this if it isn't.

File: packages/autospade/index.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  ]),
+  new Worker("assistant2", "a$$b0T!2", [
+    { type: "familiar", name: "Mosquito" },
+  ]),
+  new Worker("Zorax the Questionable", "aenimus!", [
+    { type: "skill", name: "Double-Fisted Skull Smashing" },
+    { type: "familiar", name: "Ghost of Crimbo Commerce" },
+    { type: "familiar", name: "Mosquito" },
+  ]),
+];
```
[the remark this answers is not in the document]
Comment (gausie) [reply packages/autospade/index.ts:16]: oh shit

[the remark this answers is not in the document]
Comment (gausie) [reply packages/autospade/index.ts:16]: well those are now changed

File: packages/oaf/src/commands/kol/spade.ts (hunk 1 above)
Comment (gausie) [reply packages/oaf/src/commands/kol/spade.ts:90]: Yes

File: packages/autospade/index.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+const itemFeatures = await compile("./features/items.feature");
+
+async function wait(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
```
[the remark this answers is not in the document]
Comment (gausie) [reply packages/autospade/index.ts:10]: Without making a shared util package that doesn't make much sense

**Pull request merged**